jdbc-handler/src/test/java/org/apache/hive/storage/jdbc/TestJdbcInputFormat.java

```diff
@@ -45,20 +45,33 @@
 import static org.junit.Assert.assertTrue;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.Mockito.mockStatic;
 import static org.mockito.Mockito.when;
 
-@RunWith(PowerMockRunner.class)
-@PrepareForTest(DatabaseAccessorFactory.class)
+@RunWith(MockitoJUnitRunner.class)
 public class TestJdbcInputFormat {
 
   @Mock
   private DatabaseAccessor mockDatabaseAccessor;
+  private MockedStatic<DatabaseAccessorFactory> hiveDatabaseAccessorFactory;
+
+  @Before
+  public void setup() throws Exception{
+
+    hiveDatabaseAccessorFactory = mockStatic(DatabaseAccessorFactory.class);
+    hiveDatabaseAccessorFactory.when(() -> DatabaseAccessorFactory.getAccessor(any(Configuration.class))).thenReturn(mockDatabaseAccessor);
```

Comment:
line length allowed is 120 only
https://github.com/apache/hive/blob/master/checkstyle/checkstyle.xml#L160

Reply:
fixed